pages/common/roll.md

```diff
@@ -0,0 +1,27 @@
+# roll
+
+> Rolls a user-defined dice sequence.
+
+- Roll 3 6-sided dices and sums the results:
```

Comment:
Isn't the singular of dice "die"?

Reply:
Checked back on this and you are right:
One **die**
Multiple **dice**
**dices** is a verb if I understood correctly (again something learned ^^)

Will change accordingly